plugins/inputs/supervisor/README.md

```diff
@@ -0,0 +1,87 @@
+# Supervisor Input Plugin
+
+This plugin gather information about processes that running under supervisor using XML-RPC API
+
+Plugin minimum tested version: 1.17.0
+
+### Supervisor configuration
+
+This plugin needs TCP HTTP server to be enabled without basic auth for collecting information. Here is example of
+`inet_http_server` section in supervisor config that will work with default plugin configuration.
```

Comment:
The [very large warning](http://supervisord.org/configuration.html?highlight=http#inet-http-server-section-settings) in the docs about this has me a bit concerned. No auth, means any user or process could hit these endpoints and control the system.

At the very least using auth should be required.

Reply:
When i worked on this plugin i had near zero experience with golang, so when i tried some very obvious approaches to enable auth, that didn't work, i postponed it for better times. No i feel more confident in my skills, so i try to do this right way.